repository-data/application/src/main/resources/hcm-content/taxonomies.yaml

```diff
@@ -0,0 +1,2836 @@
+      /allied_health_professionals:
+        jcr:primaryType: hippotaxonomy:category
+        hippotaxonomy:key: allied_health_professionals
+        locale.en.name: Allied health professionals
+        /art_therapist_x002f_art_psychotherapist:
+          jcr:primaryType: hippotaxonomy:category
+          hippotaxonomy:key: art_therapist_x002f_art_psychotherapist
+          locale.en.name: Art therapist/art psychotherapist
+        /diagnostic_radiographer:
+          jcr:primaryType: hippotaxonomy:category
+          hippotaxonomy:key: diagnostic_radiographer
+          locale.en.name: Diagnostic radiographer
+        /dietitian:
+          jcr:primaryType: hippotaxonomy:category
+          hippotaxonomy:key: dietitian
+          locale.en.name: Dietitian
+        /dramatherapist:
+          jcr:primaryType: hippotaxonomy:category
+          hippotaxonomy:key: dramatherapist
+          locale.en.name: Dramatherapist
+        /music_therapist:
+          jcr:primaryType: hippotaxonomy:category
+          hippotaxonomy:key: music_therapist
+          locale.en.name: Music therapist
+        /occupational_therapist:
+          jcr:primaryType: hippotaxonomy:category
+          hippotaxonomy:key: occupational_therapist
+          locale.en.name: Occupational therapist
+        /operating_department_practitioner:
+          jcr:primaryType: hippotaxonomy:category
+          hippotaxonomy:key: operating_department_practitioner
+          locale.en.name: Operating department practitioner
+        /orthoptist:
+          jcr:primaryType: hippotaxonomy:category
+          hippotaxonomy:key: orthoptist
+          locale.en.name: Orthoptist
+        /osteopath:
+          jcr:primaryType: hippotaxonomy:category
+          hippotaxonomy:key: osteopath
+          locale.en.name: Osteopath
+        /physiotherapist:
+          jcr:primaryType: hippotaxonomy:category
+          hippotaxonomy:key: physiotherapist
+          locale.en.name: Physiotherapist
+        /podiatrist:
+          jcr:primaryType: hippotaxonomy:category
+          hippotaxonomy:key: podiatrist
+          locale.en.name: Podiatrist
+        /prosthetist_x002f_orthotist:
+          jcr:primaryType: hippotaxonomy:category
+          hippotaxonomy:key: prosthetist_x002f_orthotist
+          locale.en.name: Prosthetist/orthotist
+        /speech_and_language_therapist:
+          jcr:primaryType: hippotaxonomy:category
+          hippotaxonomy:key: speech_and_language_therapist
+          locale.en.name: Speech and language therapist
+        /therapeutic_radiographer:
+          jcr:primaryType: hippotaxonomy:category
+          hippotaxonomy:key: therapeutic_radiographer
+          locale.en.name: Therapeutic radiographer
+      /ambulance_service_team:
+        jcr:primaryType: hippotaxonomy:category
+        hippotaxonomy:key: ambulance_service_team
+        locale.en.name: Ambulance service team
+      /ambulance_care_assistant_and_patient_transport_service_(pts)_driver:
```

Comment:
Can we remove brackets here ?

Reply:
Going through and changing all of these now